test/test_jit.py

```diff
@@ -5247,7 +5241,7 @@ def func(alpha, beta, x, y):
         # NOTE: cannot optimize yet because broadcasts are not inserted before the fuser runs
         self.checkScript(func, [alpha, beta, x, y], optimize=False)
 
-    @unittest.skipIf(GRAPH_EXECUTOR != ProfilingMode.PROFILING, "skip if profiling isn't enabled")
+    @unittest.skip("bailouts are being deprecated")
```

Comment:
At some point we should just delete these tests, but it probably makes sense to delete it in the same PR that rermoves Bailouts, so we get a clean git history...